### PR TITLE
Transfer gettext(f) changes to module

### PR DESCRIPTION
**Pull request by FransMeerhoff**

### CODE REVIEW

File: R/equivalencebayesianindependentsamplesttest.R [hunk 1]
```diff
@@ -152,7 +152,10 @@ EquivalenceBayesianIndependentSamplesTTest <- function(jaspResults, dataset, opt
   if (ready)
     equivalenceBayesianIndTTestTable$setExpectedSize(length(options$variables))
   
-  equivalenceBayesianIndTTestTable$addFootnote(.equivalenceGetIntervalMessage(options$lowerbound, options$upperbound))
+  message <- gettextf("I ranges from %1$s to %2$s",
```
Comment (TimKDJ) [inline R/equivalencebayesianindependentsamplesttest.R:155]: I don't think these changes are correct.
https://github.com/jasp-stats/jaspEquivalenceTTests/commit/7457abeb09c8a0cfee4bf31178347981206440c8 made sure to remove the duplication of these lines

Comment (FransMeerhoff) [reply R/equivalencebayesianindependentsamplesttest.R:155]: @TimKDJ Ah I see, I was boobytrapped. These fixes were only done on the jaspEquivalence Module repo not in stable. 
I do not think it is necessary to backport them to stable so will I close this PR?

Comment (TimKDJ) [reply R/equivalencebayesianindependentsamplesttest.R:155]: I'm not sure if we want the table renaming though, @Jillderon?

Comment (Jillderon) [reply R/equivalencebayesianindependentsamplesttest.R:155]: No changes are not needed, this PR can be closed!

**Pull request closed without merging**